lib/adb.js

```diff
@@ -193,6 +193,15 @@ ADB.prototype.shell = function (cmd, cb) {
   this.exec(execCmd, cb);
 };
 
+// This is fix for https://github.com/appium/appium/issues/5396 getting the devices propery when runnning with mulitiple appium instance
+ADB.prototype.shellForgettingDeviceProperty = function (cmd, cb) {
```

Comment:
`shellForgettingDeviceProperty` ought to be `shellForGettingDeviceProperty` (i.e., "for getting" is different than "forgetting").